.github/workflows/build_and_test_and_release.yml

```diff
@@ -20,10 +20,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: "recursive"
-      - name: Setup Python 3.8
+      - name: Setup Python 3.9
```

Comment:
```suggestion
      - name: Setup Python 3.8
```

Reply:
Let's keep it at 3.8 for now in tests to make sure we're still compatible with 3.8, given that https://github.com/splitgraph/sgr/pull/788 has resolved the issues that led us to think we had to drop support for 3.8 everywhere.